reports/vamshi99.md

```diff
@@ -0,0 +1,209 @@
+   - details: >
+      Auto assign issue based on commit mention in PR
+ - 6:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/0eec2ec32e7afd7d017cad4c67a741c688e1bfc4
+   - details: >
+      apps.py: Add helper function to fetch plugin settings
+ - 7:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/e2224d7d1f61a34065f741535e53e676dc51ff4b
+   - details: >
+      Add Team class
+ - 8:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/f9d08c9a88c0b958a4c2ec5eb813fcd124b5300e
+   - details: >
+      Mark PR as WIP if coala plugin detects errors
+ - 9:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/c4abdd4f9367af8ce73f39f525383def68de6b62
+   - details: >
+      MergeRequest: Add mergeable property
+ - 10:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/db76142d3ecb1e6cd9cd10885ae2475cb07c4d3d
+   - details: >
+      Notify merge conflicts in a PR
+ - 11:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/ad5480c9b0a124d98020a4ad3fd3caf20fa1e321
+   - details: >
+      pr_rejector: Support regex in branch names
+ - 12:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/94735ff0f1024d6f293db90516087d8e4d2c353d
+   - details: >
+      Organization: Add issues property and filter_issues method
+ - 13:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/fff0d568f1f2731c5055995ff19bbc93be842d5f
+   - details: >
+      Repository: Filter issues by label, assignee
+ - 14:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/c025311167b07da5e825433a4d4d5e376adfed62
+   - details: >
+      GitLabUser: Fetch user id from username
+ - 15:
+   - repo: mr-rebaser
+   - link: https://gitlab.com/gitmate/open-source/mr-rebaser/commit/fad8305823a1058e93cd2397b5239a882b66c3f9
+   - details: >
+      Add squash pull request command
+ - 16:
+   - repo: mr-rebaser
+   - link: https://gitlab.com/gitmate/open-source/mr-rebaser/commit/3124c07ca5cd46acf985dc25fe22889e5a4744d0
+   - details: >
+      Dockerfile: Install requirements
+ - 17:
+   - repo: mr-rebaser
+   - link: https://gitlab.com/gitmate/open-source/mr-rebaser/commit/68360f3f0fac686907c7eb44c1a70bfa9e276d42
+   - details: >
+      run.py: Update squash command
+ - 18:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/commit/dda98a8ab67d01376d3199bb82260838cebab7b0
+   - details: >
+      rebaser: Add squash command
+ - 19:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/f4ec3c602745bcf08405479a684876cc1080a21f
+   - details: >
+      Issue: Add available_assignees property
+ - 20:
+   - repo: gitmate-2
+   - link: https://gitlab.com/gitmate/open-source/gitmate-2/merge_requests/525
```

Comment:
give the current sha of the MR, I suggest doing this by using the `/diffs` page for the top commit.

Also ... the pipeline is failing.... we are we accepting that?

the `details:` below needs to say "WIP" at least.

And a longer note is needed in the report below explain why.

Reply:
That MR is approved and pipeline is failing due to some change in dependency repo(which I couldn't access). As of now I will submit the current SHA.

Reply:
@jayvdb we have some internal issues with submodules.
I guess @yukiisbored is looking after it.

Reply:
Ok, great, can you add a note on https://gitlab.com/gitmate/open-source/gitmate-2/merge_requests/525 that the pipeline is failing because of environmental issues that are the resposibility of devops to fix, and that it (sha link) is approved and will be merged once that has been fixed by devops.